code/widgets/org.eclipse.scout.widgets.shared/src/main/java/org/eclipse/scout/widgets/shared/services/code/SeverityCodeType.java

```diff
@@ -27,6 +27,11 @@ protected String getConfiguredText() {
     return TEXTS.get("SeverityType");
   }
 
+  @Override
+  protected String getConfiguredTextPlural() {
+    return TEXTS.get("SeverityTypes");
```

Comment:
Use "Severity" for both singular and plural? Text keys for "SeverityType" and "SeverityTypes" seem to be missing(?)